mock-server/public/css/variables.css

```diff
@@ -1,296 +1,299 @@
 :root {
-  /* sizes*/
-  --header-size: 52px;
-  /* colors */
-	--material-color-red: #f44336;
-	--material-color-red-50: #ffebee;
-	--material-color-red-100: #ffcdd2;
-	--material-color-red-200: #ef9a9a;
-	--material-color-red-300: #e57373;
-	--material-color-red-400: #ef5350;
-	--material-color-red-500: var(--material-color-red);
-	--material-color-red-600: #e53935;
-	--material-color-red-700: #d32f2f;
-	--material-color-red-800: #c62828;
-	--material-color-red-900: #b71c1c;
-	--material-color-red-a100: #ff8a80;
-	--material-color-red-a200: #ff5252;
-	--material-color-red-a400: #ff1744;
-	--material-color-red-a700: #d50000;
+    /* app specific */
+    --footer-wave-desktop: url(https://res.cloudinary.com/glovoapp/image/fetch/f_svg,q_auto:low/https://glovoapp.com/images/landing/footer-wave-desktop.svg);
+    --footer-wave-mobile: url(https://res.cloudinary.com/glovoapp/image/fetch/f_svg,q_auto:low/https://glovoapp.com/images/landing/footer-wave-mobile.svg);
```

Comment:
These 2 vars got lost due to JSX cleanup, bringing them back to this file instead.
The rest is formatting noise.